src/options.cpp

```diff
@@ -1320,6 +1322,31 @@ void show_options(bool ingame)
             g->init_ui();
             if( ingame ) {
                 g->refresh_all();
+
+                DebugLog( D_INFO, DC_ALL ) << "Loaded tileset: " << OPTIONS["TILES"].getValue();
+
+                tile_loading_report(termap, "Terrain", "");
+                tile_loading_report(furnmap, "Furniture", "");
+                tile_loading_report(Item_factory::Item_factory().get_all_itypes(), "Items", "");
+                tile_loading_report(MonsterGenerator::generator().get_all_mtypes(), "Monsters", "");
```

Comment:
If that other places wants a copy, it will just make the copy from the returned reference instead. But as I said, that single copy here is probably no real problem.

> I wish there was a global instance of this map like furnmap and termap.

There is a global instance of that map, it's encapsulated in `MonsterGenerator`. The global `termap` for example can be *changed* (accidentally or not) at any place by anybody which would totally screw up the game.

For example I could write `termap.erase(termap.begin())` at any place and suddenly a terrain type is gone. Encapsulating those maps and proving only const access prevents this kind of error.